src/main/java/persistence/sql/ddl/DdlGenerator.java

```diff
@@ -28,17 +32,46 @@ private String generateColumnsClause(final EntityMetadata<?> entityMetadata) {
         final StringBuilder builder = new StringBuilder();
         builder.append("(");
 
-        entityMetadata.getColumns().forEach(column ->
-                builder.append(generateColumnDefinition(column))
-                        .append(",")
+        entityMetadata.getColumns().forEach(column -> {
+                    if (column.isOneToMany()) {
+                        return;
+                    }
+                    builder.append(generateColumnDefinition(column))
+                            .append(",");
+                }
         );
 
-        builder.append(generatePKConstraintClause(entityMetadata));
+        builder.append(generateAssociatedColumnsClause(entityMetadata));
 
+        builder.append(generatePKConstraintClause(entityMetadata));
         builder.append(")");
         return builder.toString();
     }
 
+    private String generateAssociatedColumnsClause(final EntityMetadata<?> entityMetadata) {
+        final StringBuilder builder = new StringBuilder();
+        final Set<EntityMetadata<?>> allAssociatedEntitiesMetadata = EntityMetadataProvider.getInstance().getAllAssociatedEntitiesMetadata(entityMetadata);
+        allAssociatedEntitiesMetadata.forEach(associatedEntityMetadata -> {
+            final EntityIdColumn associatedEntityIdColumn = associatedEntityMetadata.getIdColumn();
+            associatedEntityMetadata.getOneToManyColumns().forEach(entityOneToManyColumn ->
+                    builder.append(entityOneToManyColumn.getName())
+                            .append(" ")
+                            .append(dialect.getColumnTypeMapper().getColumnName(associatedEntityIdColumn.getType()))
```

Comment:
```suggestion
                            .append(dialect.getColumnTypeMapper().getColumnName(associatedEntityMetadata.columnType()))
```

컬럼을 가지고 있는 `associatedEntityMetadata`에게 물어보면 어떨까요?

Reply:
[`823de3b`](https://github.com/next-step/jpa-association/pull/18/commits/823de3bd1f31d9f3f32b3e009e5240f33473f6e7) - 반영완료